contrib/sssd.spec.in

```diff
@@ -804,6 +808,10 @@ done
 %{_datadir}/systemtap/tapset/sssd.stp
 %{_datadir}/systemtap/tapset/sssd_functions.stp
 %{_mandir}/man5/sssd-systemtap.5*
+%if 0%{?rhel} >= 9 || 0%{?fedora}
```

Comment:
Either I misunderstand something, or this condition is inconsistent:
 - `install` above is unconditional.
 - config is packages for  rhel >= 9 and fedora
 - users below are created for rhel only (any version)

Frankly I wouldn't touch RHEL8 and RHEL9 and only bring this to F40+/RHEL10